.env

```diff
@@ -1,3 +1,3 @@
 PORT=3000
-
+LDAP_URL = "http://hslinux:38383/api/v1/auth"
```

Comment:
Don't commit the .env file. I only did in the template bc i don't know how to automate the template including one, without having it just in the repo.